client-src/elements/chromedash-feature-page.ts

```diff
@@ -420,6 +491,18 @@ export class ChromedashFeaturePage extends LitElement {
         </div>
       `);
     }
+    if (this.showAccuracyWarning()) {
+      warnings.push(html`
+        <div class="warning">
+          Your feature is oudated, but slated to launch soon. Please click the
+          link below to update your feature entry.
+          <br />
+          <a href="/guide/verify_accuracy/${this.featureId}"
+            >Verify feature accuracy</a
+          >
```

Comment:
For feature owners, how about something like:

```suggestion
          Your feature hasn't been verified as accurate since <sl-relative-time date=${this.feature.accurate_as_of}></sl-relative-time>, but it
          ${shipDate(closestShippingMilestone) > this.currentDate ? 'is scheduled to ship' : 'claims to have shipped'}
          <sl-relative-time date=${shipDate(closestShippingMilestone)}></sl-relative-time>
          Please <a href="/guide/verify_accuracy/${this.featureId}"
            >verify that your feature is accurate</a
          >.
```

`closestShippingMilestone` should be the greatest shipping milestone that's `<= latestStableVersion + 2`, I think?

Reply:
I will move 'claims to have shipped' to the follow-up PR as mentioned in the reply above.